R/span.R

```diff
@@ -0,0 +1,101 @@
+#' Calculate time span between dates
+#'
+#' @param data The input data frame or linelist
+#' @param target_column A string used to specify the name of the date column of
+#'    interest. The values in this columns should be of type 'Date' in ISO
+#'    format ("2024-01-31").
+#' @param end_date An end date. It can be either a character that is the name of
+#'    another column of type 'Date' from the input data or a vector of Dates or
+#'    a single Date value. This should also be in the ISO format ("2024-01-31").
+#'    Default is today's date `Sys.Date()`.
+#' @param span_unit A string that specifies the units in which the spanning time
```

Comment:
For consistency, consider changing `spanning time` to `time span` on this line and the 4 other occurences on line 14, 15 and 17

Reply:
fixed in commit `71ff418`